src/Module.php

```diff
@@ -11,5 +11,9 @@
  */
 final class Module extends \luya\base\Module implements CoreModuleInterface
 {
-
+    /**
+     * @var boolean Whether hidden page should be included or not, defaults to false in order to make sure only
+     * visible pages are included by default. In order to include hidden pages into the sitemap enable $withHidden.
+     */
+    public $withHidden = false;
```

Comment:
I was not sure how to name the property, other options could be:

+ withHidden
+ ignoreHidden
+ showHidden
+ includeHidden

Reply:
`$withHidden` sounds fine to me